packages/core/src/network.js

```diff
@@ -87,6 +89,25 @@ export class Network {
     });
   }
 
+  // Validates that requestId is still valid as sometimes request gets cancelled and we have already executed
+  // _forgetRequest for the same, but we still attempt to make a call for it and it fails
+  // with Protocol error (Fetch.failRequest): Invalid InterceptionId.
+  async send(session, method, params) {
+    /* istanbul ignore else: currently all send have requestId */
+    if (params.requestId) {
+      /* istanbul ignore if: race condition, very hard to mock this */
+      if (this.isAborted(params.requestId)) {
+        throw new Error(ABORTED_MESSAGE);
+      }
+    }
```

Comment:
what if the requestId is not present?

Reply:
there is a check on line 97, not sure if you are referring to anything else

Reply:
that check is for if the requestId is present, if it's not present are we expecting to hit the condition for "invalid inteception id"?